string[] meshNames
string[] meshURLs
---
int32[] errcodes
string[] errmsgs
